jupyter_scheduler/orm.py

```diff
@@ -100,20 +103,47 @@ class Job(CommonColumns, Base):
     url = Column(String(256), default=generate_jobs_url)
     pid = Column(Integer)
     idempotency_token = Column(String(256))
+    # All new columns added to this table must be nullable to ensure compatibility during database migrations.
+    # Any default values specified for new columns will be ignored during the migration process.
 
 
 class JobDefinition(CommonColumns, Base):
     __tablename__ = "job_definitions"
+    __table_args__ = {"extend_existing": True}
     job_definition_id = Column(String(36), primary_key=True, default=generate_uuid)
     schedule = Column(String(256))
     timezone = Column(String(36))
     url = Column(String(256), default=generate_job_definitions_url)
     create_time = Column(Integer, default=get_utc_timestamp)
     active = Column(Boolean, default=True)
+    # All new columns added to this table must be nullable to ensure compatibility during database migrations.
+    # Any default values specified for new columns will be ignored during the migration process.
 
 
-def create_tables(db_url, drop_tables=False):
+def update_db_schema(engine, Base):
+    inspector = inspect(engine)
+
+    with engine.connect() as connection:
```

Comment:
nit: Rather than wrapping the logic with the connection, it is cleaner to collect all the alter statements and execute this separately after this in a separate block.
```python
# Rest of the logic here

with engine.connect() as connection:
    for alter_statement in alter_statements:
        connection.execute(alter_statement)
```

Reply:
Good point, it is cleaner this way. I also added a check to ensure that `alter_statements` is not empty before instantiating the connection so this would also optimize a number of db connections.